bundled/boost-1.70.0/include/boost/signals2/detail/auto_buffer.hpp

```diff
@@ -142,7 +142,7 @@ namespace detail
         typedef typename Allocator::size_type            size_type;
         typedef typename Allocator::difference_type      difference_type;
         typedef T*                                       pointer;
-        typedef typename Allocator::pointer              allocator_pointer;
+        typedef typename Allocator::value_type*          allocator_pointer;
```

Comment:
Does this match recent upstream changes in boost?

Reply:
I should have looked at that first. They do it slightly differently and I changed it accordingly.